src/graphql-server/schema.graphql

```diff
@@ -68,8 +68,11 @@ type Transaction {
   presentBalance: Float
   notes: String
   happName: String
-  isPayingARequest: Boolean
   canceledBy: HolofuelUser
+  isPayingARequest: Boolean
+  inProcess: Boolean
+  actioned: Boolean
+  stale: Boolean
```

Comment:
I would call all these booleans `isSomething`, to make it clear they're booleans